exe/facter.cc

```diff
@@ -122,25 +124,25 @@ int main(int argc, char **argv)
         // Keep this list sorted alphabetically
         po::options_description visible_options("");
         visible_options.add_options()
-            ("color", "Enables color output.")
-            ("config,c", po::value<string>(&conf_dir), "Specify the location of the config file.")
-            ("custom-dir", po::value<vector<string>>(&custom_directories), "A directory to use for custom facts.")
-            ("debug,d", po::bool_switch()->default_value(false), "Enable debug output.")
-            ("external-dir", po::value<vector<string>>(&external_directories), "A directory to use for external facts.")
-            ("help,h", "Print this help message.")
-            ("json,j", "Output in JSON format.")
-            ("show-legacy", "Show legacy facts when querying all facts.")
-            ("log-level,l", po::value<level>()->default_value(level::warning, "warn"), "Set logging level.\nSupported levels are: none, trace, debug, info, warn, error, and fatal.")
-            ("no-color", "Disables color output.")
-            ("no-custom-facts", po::bool_switch()->default_value(false), "Disables custom facts.")
-            ("no-external-facts", po::bool_switch()->default_value(false), "Disables external facts.")
-            ("no-ruby", po::bool_switch()->default_value(false), "Disables loading Ruby, facts requiring Ruby, and custom facts.")
-            ("puppet,p", "(Deprecated: use `puppet facts` instead) Load the Puppet libraries, thus allowing Facter to load Puppet-specific facts.")
-            ("trace", po::bool_switch()->default_value(false), "Enable backtraces for custom facts.")
-            ("verbose", po::bool_switch()->default_value(false), "Enable verbose (info) output.")
-            ("version,v", "Print the version and exit.")
-            ("yaml,y", "Output in YAML format.")
-            ("strict", "Enables more aggressive error reporting.");
+            ("color", _("Enables color output.").c_str())
```

Comment:
I'll take a look at setup_logging. We may want to catch and report errors in leatherman.